ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -249,6 +258,103 @@ export default {
         },
       },
     },
+    {
+      id: 'DelubrumSav Guard Fiery Portent',
+      netRegex: NetRegexes.startsUsing({ source: 'Queen\'s Soldier', id: '583F', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Soldat Der Königin', id: '583F', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Soldat De La Reine', id: '583F', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'クイーンズ・ソルジャー', id: '583F', capture: false }),
+      delaySeconds: 4,
```

Comment:
Why the delaySeconds on this? Maybe better to have the early callout + durationSeconds if it needs to stay up for longer? Same question on the icy portent below.

Reply:
The cast is 6 seconds long, and so I figure a cast on startsUsing is too early. To me, 3 second delay still felt too early and 4 seconds feels too short.

This does need a durationSeconds though. The Pyretic (3C0) debuff lasts 3 seconds.

Reply:
Sounds reasonable! I was just wondering if it was better to just call this out as early as possible.

Reply:
The updated timings (delaySeconds of 3.5) lined up with the my group's manual callouts, so felt good now.

Reply:
What would you think of modifying `delaySeconds` to take a function of the cast duration instead? As `delaySeconds: (data, matches) => parseFloat(matches.duration) - 2,`. It's easier to see exactly how much advance notification we're giving the user when it's written this way.

Reply:
Good idea @JLGarber I have added this change.

Reply:
FYI, not moving during Icy Portent causes Deep Freeze '4E6'. Also the behavior and debuffs are the same in The Queen.